app/oracle.py

```diff
@@ -101,15 +103,7 @@
 
 ORACLE_FROM_BLOCK = int(os.getenv('ORACLE_FROM_BLOCK', 0))
 
-if eth1_provider.startswith('http'):
-    provider = HTTPProvider(eth1_provider)
-elif eth1_provider.startswith('ws'):
-    provider = WebsocketProvider(eth1_provider)
-else:
-    logging.error('Unsupported ETH provider!')
-    exit(1)
-
-w3 = Web3(provider)
+w3 = Web3(MultiProvider(eth1_provider.split(',')))
```

Comment:
I can also rename WEB3_PROVIDER_URI -> WEB3_RPC_ENDPOINTS
What do you think?

Reply:
I suggest for using just "Provider" name with []rpc_endpoints signature.
Provider([]rpc_endpoints signature), and if it's possiblie make annottation for "Provider" like this.

```
@endpoints - an array with bla...bla
@returns bla bla bla
func Provider(endpoints []rpc_endpoints signature): Provider
```

Reply:
Function overloading is not nice to do in python.